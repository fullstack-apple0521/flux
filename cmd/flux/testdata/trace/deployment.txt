
Object:         deployment/podinfo
Namespace:      podinfo
Status:         Managed by Flux
---
HelmRelease:    podinfo
Namespace:      podinfo
Revision:       6.0.0
Status:         Last reconciled at 2021-07-16 15:42:20 +0000 UTC
Message:        Release reconciliation succeeded
---
HelmChart:      podinfo-podinfo
Namespace:      flux-system
Chart:          podinfo
Version:        6.0.0
Revision:       6.0.0
Status:         Last reconciled at 2021-07-16 15:32:09 +0000 UTC
Message:        Fetched revision: 6.0.0
---
HelmRepository: podinfo
Namespace:      flux-system
URL:            https://stefanprodan.github.io/podinfo
Revision:       8411f23d07d3701f0e96e7d9e503b7936d7e1d56
Status:         Last reconciled at 2021-07-11 00:25:46 +0000 UTC
Message:        Fetched revision: 8411f23d07d3701f0e96e7d9e503b7936d7e1d56

