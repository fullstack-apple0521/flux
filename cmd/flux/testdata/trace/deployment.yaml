---
apiVersion: v1
kind: Namespace
metadata:
  name: {{ .fluxns }}
---
apiVersion: v1
kind: Namespace
metadata:
  name: {{ .ns }}
---
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app.kubernetes.io/name: podinfo
    app.kubernetes.io/managed-by: Helm
    helm.toolkit.fluxcd.io/name: podinfo
    helm.toolkit.fluxcd.io/namespace: {{ .ns }}
  name: podinfo
  namespace: {{ .ns }}
spec:
  replicas: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: podinfo
  template:
    metadata:
      labels:
        app.kubernetes.io/name: podinfo
    spec:
      containers:
      - name: hello
        command: [ "echo hello world" ]
        image: busybox
---
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: infrastructure
    kustomize.toolkit.fluxcd.io/namespace: {{ .fluxns }}
  name: podinfo
  namespace: {{ .ns }}
spec:
  chart:
    spec:
      chart: podinfo
      sourceRef:
        kind: HelmRepository
        name: podinfo
        namespace: {{ .fluxns }}
  interval: 5m
status:
  conditions:
  - lastTransitionTime: "2021-07-16T15:42:20Z"
    message: Release reconciliation succeeded
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
  helmChart: {{ .fluxns }}/podinfo-podinfo
  lastAppliedRevision: 6.0.0
  lastAttemptedRevision: 6.0.0
  lastAttemptedValuesChecksum: c31db75d05b7515eba2eef47bd71038c74b2e531
---
apiVersion: source.toolkit.fluxcd.io/v1beta1
kind: HelmChart
metadata:
  name: podinfo-podinfo
  namespace: {{ .fluxns }}
spec:
  chart: podinfo
  sourceRef:
    kind: HelmRepository
    name: podinfo
  version: 6.0.0
  interval: 5m
status:
  artifact:
    checksum: cf13ba96773d9a879cd052c86e73199b3f96c854
    lastUpdateTime: "2021-08-01T04:42:55Z"
    revision: 6.0.0
    path: "example"
    url: "example"
  conditions:
  - lastTransitionTime: "2021-07-16T15:32:09Z"
    message: 'Fetched revision: 6.0.0'
    reason: ChartPullSucceeded
    status: "True"
    type: Ready

---
apiVersion: source.toolkit.fluxcd.io/v1beta1
kind: HelmRepository
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: infrastructure
    kustomize.toolkit.fluxcd.io/namespace: flux-system
  name: podinfo
  namespace: {{ .fluxns }}
spec:
  interval: 5m
  timeout: 1m0s
  url: https://stefanprodan.github.io/podinfo
status:
  artifact:
    checksum: 8411f23d07d3701f0e96e7d9e503b7936d7e1d56
    lastUpdateTime: "2021-07-11T00:25:46Z"
    revision: sha1:8411f23d07d3701f0e96e7d9e503b7936d7e1d56
    path: "example"
    url: "example"
  conditions:
  - lastTransitionTime: "2021-07-11T00:25:46Z"
    message: 'Fetched revision: main@sha1:8411f23d07d3701f0e96e7d9e503b7936d7e1d56'
    reason: IndexationSucceed
    status: "True"
    type: Ready
---
apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
kind: Kustomization
metadata:
  name: infrastructure
  namespace: {{ .fluxns }}
spec:
  path: ./infrastructure/
  sourceRef:
    kind: GitRepository
    name: flux-system
  validation: client
  interval: 5m
  prune: true
status:
  conditions:
  - lastTransitionTime: "2021-08-01T04:52:56Z"
    message: 'Applied revision: main@sha1:696f056df216eea4f9401adbee0ff744d4df390f'
    reason: ReconciliationSucceeded
    status: "True"
    type: Ready
---
apiVersion: source.toolkit.fluxcd.io/v1beta1
kind: GitRepository
metadata:
  labels:
    kustomize.toolkit.fluxcd.io/name: flux-system
    kustomize.toolkit.fluxcd.io/namespace: {{ .fluxns }}
  name: flux-system
  namespace: {{ .fluxns }}
spec:
  gitImplementation: go-git
  ref:
    branch: main
  secretRef:
    name: flux-system
  interval: 5m
  url: ssh://git@github.com/example/repo
