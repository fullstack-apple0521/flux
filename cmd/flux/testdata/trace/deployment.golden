
Object:         deployment/podinfo
Namespace:      {{ .ns }}
Status:         Managed by Flux
---
HelmRelease:    podinfo
Namespace:      {{ .ns }}
Revision:       6.0.0
Status:         Last reconciled at {{ .helmReleaseLastReconcile }}
Message:        Release reconciliation succeeded
---
HelmChart:      podinfo-podinfo
Namespace:      {{ .fluxns }}
Chart:          podinfo
Version:        6.0.0
Revision:       6.0.0
Status:         Last reconciled at {{ .helmChartLastReconcile }}
Message:        Fetched revision: 6.0.0
---
HelmRepository: podinfo
Namespace:      {{ .fluxns }}
URL:            https://stefanprodan.github.io/podinfo
Revision:       8411f23d07d3701f0e96e7d9e503b7936d7e1d56
Status:         Last reconciled at {{ .helmRepositoryLastReconcile }}
Message:        Fetched revision: 8411f23d07d3701f0e96e7d9e503b7936d7e1d56
